chef/cookbooks/cpe_choco/resources/cpe_choco_apps.rb

```diff
@@ -94,15 +94,17 @@ def provider_supported_opts(options, pkg)
   def action_guard(option)
     if option['version'].casecmp('latest').zero?
       :upgrade
```

Comment:
Thanks, Good Catch... This is embarrassing we didnt see this. 

I think a simpler solution would be to add a return, 
```suggestion
      return :upgrade
```